packages/client/lib/commands/COMMAND_DOCS.ts

```diff
@@ -0,0 +1,38 @@
+import { RedisCommandArguments } from '.';
+import { pushVerdictArguments } from './generic-transformers';
+
+export const IS_READ_ONLY = true;
+
+export function transformArguments(keys: string | Array<string>): RedisCommandArguments {
+    return pushVerdictArguments(['COMMAND', 'DOCS'], keys);
+}
+
+type CommandDocumentation = {
+    summary: string;
+    since: string;
+    group: string;
+    complexity: string;
+    history?: Array<[version: string, feature: string]>;
+};
+
+type CommandDocsReply = Array<[CommandName: string, CommandDocumentation: CommandDocumentation]>;
```

Comment:
Please change the type (and update the code accordingly) to:
```typescript
type CommandDocsReply = {
  [command: string]: {
    summary: string;
    since: string;
    group: string;
    complexity: string;
    history?: Array<{
      version: string;
      description: string;
    }>;
  };
};
```

In general, try to respect RESP3 ([`HELLO 3`](https://redis.io/commands/hello/)) types as much as possible.